### ISSUE TITLE
Importing Assets Checked Out To Users

### ISSUE THREAD

**Issue by ACurnow** [snipe/snipe-it#7176]
#### Please confirm you have done the following before posting your bug report:

- [x] I have enabled debug mode 
- [x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
When importing assets to check out automatically to users, CSV import hangs on Web Importer and presents error using Shell command.
Command:
`php artisan snipeit:import C:\Users\Administrator\Desktop\User_Assets.csv --item-type Asset --user_id 1 --verbose --update`
Error:
```
In CheckoutAssetNotification.php line 114:

  [Symfony\Component\Debug\Exception\FatalThrowableError]
  Call to a member function present() on null
```
**To Reproduce**
1. Go to Web Importer.
2. Attempt to import CSV file.
Alternatively:
1. Open Shell.
2. Run above excerpt.

**Expected behavior**
Assets imported, checked out to appropriate users.

**Screenshots**
Can provide screenshots if required.

**Server (please complete the following information):**
 - Snipe-IT Version: v4.6.17 build 4026 (g888bdbdb6)
 - OS: Windows Server 2016 Standard
 - Web Server: IIS
 - PHP Version: 7.1.10

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: 74.0.3729.169

**Smartphone (please complete the following information):**
N/A

**Error Messages**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error:
**Please see above in description**
- If a stacktrace is provided in the error, include that too.
```
Exception trace:
 () at C:\inetpub\wwwroot\snipe-it\app\Notifications\CheckoutAssetNotification.php:114
 App\Notifications\CheckoutAssetNotification->toSlack() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Notifications\Channels\SlackWebhookChannel.php:45
 Illuminate\Notifications\Channels\SlackWebhookChannel->send() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Notifications\NotificationSender.php:113
 Illuminate\Notifications\NotificationSender->sendToNotifiable() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Notifications\NotificationSender.php:89
 Illuminate\Notifications\NotificationSender->sendNow() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Notifications\NotificationSender.php:64
 Illuminate\Notifications\NotificationSender->send() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Notifications\ChannelManager.php:36
 Illuminate\Notifications\ChannelManager->send() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Notifications\RoutesNotifications.php:18
 App\Models\User->notify() at C:\inetpub\wwwroot\snipe-it\app\Models\Loggable.php:80
 App\Models\Asset->logCheckout() at C:\inetpub\wwwroot\snipe-it\app\Models\Asset.php:249
 App\Models\Asset->checkOut() at C:\inetpub\wwwroot\snipe-it\app\Importer\AssetImporter.php:117
 App\Importer\AssetImporter->createAssetIfNotExists() at C:\inetpub\wwwroot\snipe-it\app\Importer\AssetImporter.php:41
 App\Importer\AssetImporter->handle() at C:\inetpub\wwwroot\snipe-it\app\Importer\Importer.php:131
 App\Importer\Importer->App\Importer\{closure}() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Database\Concerns\ManagesTransactions.php:29
 Illuminate\Database\Connection->transaction() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php:327
 Illuminate\Database\DatabaseManager->__call() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php:221
 Illuminate\Support\Facades\Facade::__callStatic() at C:\inetpub\wwwroot\snipe-it\app\Importer\Importer.php:138
 App\Importer\Importer->import() at C:\inetpub\wwwroot\snipe-it\app\Console\Commands\ObjectImportCommand.php:83
 App\Console\Commands\ObjectImportCommand->fire() at n/a:n/a
 call_user_func_array() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:29
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:87
 Illuminate\Container\BoundMethod::callBoundMethod() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php:31
 Illuminate\Container\BoundMethod::call() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php:539
 Illuminate\Container\Container->call() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Console\Command.php:182
 Illuminate\Console\Command->execute() at C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Command\Command.php:255
 Symfony\Component\Console\Command\Command->run() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Console\Command.php:167
 Illuminate\Console\Command->run() at C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Application.php:960
 Symfony\Component\Console\Application->doRunCommand() at C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Application.php:255
 Symfony\Component\Console\Application->doRun() at C:\inetpub\wwwroot\snipe-it\vendor\symfony\console\Application.php:148
 Symfony\Component\Console\Application->run() at C:\inetpub\wwwroot\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php:122
 Illuminate\Foundation\Console\Kernel->handle() at C:\inetpub\wwwroot\snipe-it\artisan:35
```
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.

**Additional context**
- Is this a fresh install or an upgrade? **Fresh install**
- What OS and web server you're running Snipe-IT on: Please see above "Server"
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc): **Manual installation**
- Include what you've done so far in the installation, and if you got any error messages along the way: **Installation fully complete as per site Documentation**
- Indicate whether or not you've manually edited any data directly in the database **No I have not**

**Add any other context about the problem here**
I have browsed several different related issues and have included using the Full Name and Username fields on the CSV file, as well as trying to import with 5 sample assets to check folder permissions and formatting. It seems the issue is related to the act of checking out the device to a user, as every other operation besides this step runs perfectly without any issues.

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

**Comment by ACurnow** [comment]
This issue has been resolved.

Turns out that the Slack integration I had set up as a test was stopping the Asset checkout from completing because it couldn't find the users in Slack's directory (Which makes sense, since it was a test instance).

This is somewhat surprising that it doesn't just skip over the Slack communication if the user cannot be found and instead alert the Admin performing the import.
